clicommand/agent_start.go

```diff
@@ -245,6 +246,12 @@ var AgentStartCommand = cli.Command{
 			Usage:  "Path to where the builds will run from",
 			EnvVar: "BUILDKITE_BUILD_PATH",
 		},
+		cli.StringFlag{
+			Name:   "repos-path",
+			Value:  "",
+			Usage:  "Path to where the repositories will be stored",
```

Comment:
Maybe for explicitness' sake: 

```suggestion
			Usage:  "Path to where the mirrored copies of repositories will be stored",
```